server/model/config.go

```diff
@@ -1309,20 +1310,36 @@ func (s *LogSettings) SetDefaults() {
 		s.FileJson = NewBool(true)
 	}
 
+	if s.AdvancedLoggingJSON == nil {
+		s.AdvancedLoggingJSON = []byte("{}")
```

Comment:
Shouldn't we check the value of `AdvancedLoggingConfig` and copy over that to this new field? This would avoid admins having to do it manually.

Reply:
Great idea. Thanks!

Reply:
Done